src/index.js

```diff
@@ -415,12 +417,17 @@ class Dropzone extends React.Component {
       getDataTransferItems,
       ...divProps
     } = props
+    /* eslint-enable no-unused-vars */
 
+    /* eslint-disable jsx-a11y/no-static-element-interactions */
+    /* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Maybe it's easier to disable these rules at all?

Reply:
They will go away in #468